### DISCUSSION TITLE
Nach Umzug auf andere Hardware funktionieren Homematic Komponenten nicht mehr

### DISCUSSION THREAD

**Discussion by martin1673** [SukramJ/aiohomematic#1026] [Q&A] [answered]
Hallo, ich wollte auf neue Hardware umziehen (von Synology auf PI3B)
Dazu habe ich Homeassistant auf dem Pi installiert und das Backup der Installation von der Synology eingespielt.
Jetzt funktionieren einige Schalter nicht mehr, bei den Rolläden z.b. kanni ch nur den einen HM Rolladen Aktor und den HMIP Aktor steuern. die HM Jalousie Aktoren lassen sich nicht mehr steuern.
Die Lichter gehen teilweise auch nicht mehr zu steuern, ich bekomme aber immer den richtigen zustand angezeigt.
Auch wenn ich die alte Installation auf der Synology wieder hochfahre habe ich das gleiche Problem. 
Jetzt habe ich auf dem PI alles nochmal neu installiert mit dem gleichen Ergebnis.
Daher vermute ich das es evtl. in der CCU mit raspberrymatic zu suchen ist.
Hat hier jemand schon so ein Problem gehabt? 
Ich weis nicht mehr weiter

**Comment by danielperna84** [comment]
Hast du denn die CCU schon mal neu gestartet? Ich nehme mal an, dass die ein separates Gerät ist.

**Comment by martin1673** [comment]
Ja die wurde schon mehrmals neu gestartet.
Die Lichter gehen mittlerweile alle.
Nur bei den Rolläden funktioniert nicht alles. 
Was funktioniert ist der Homematic Rolladenaktor und der Homematic IP Rolladen Aktor. Die Homematic Jalousie Aktoren funktionieren nur manchmal, aber nicht von der Übersicht wenn ich im Dashboard nach oben oder unten klicke, sondern wenn dann im Gerät selbst mit dem Schieberegler, aber wie gesagt nur ab und zu. Im Dashboard kann ich am Symbol auch sehen wenn sie sich gerade bewegen ( ausgelöst durch das Programm auf der Raspberrymatic) wie gesagt vor dem Umzug ging das

**Reply by danielperna84** [reply]
Das klingt dann schon sehr CCU-spezifisch. Da ist das Anliegen glaube ich im Homematic Forum besser aufgehoben. Die Integration macht ja anscheinend was sie soll.  Warum das durch den Umzug ausgelöst wurde bleibt offen. Wie sieht's denn mit dem Duty Cycle aus?

**Reply by danielperna84** [reply, marked as answer]
Nachtrag: ist das hier vielleicht dein Problem? https://github.com/danielperna84/hahomematic/issues/1027

**Comment by martin1673** [comment]
Den Post werde ich später mal prüfen

**Reply by martin1673** [reply]
Ja das wars... geht wieder... da ist das Update mit dem Umzug zusammen gefallen.
Danke für den Tip
